### PR TITLE
create quick guide button to pop up modal

### PR DESCRIPTION
**Pull request by bthaile**
Created quick guide button to pop up modal.

image

### CODE REVIEW

File: packages/augur-ui/src/modules/reporting/common.styles.less [hunk 1]
```diff
@@ -13,3 +13,17 @@
...
+  background-color: @color-secondary-dark;
+  border: 1px solid @color-alternate-info-text;
+  color: @color-alternate-info-text;
+  display: flex;
+  justify-content: center;
+  line-height: @size-16;
+  padding: 0.5rem;
+  text-align: center;
+  width: 100%;
+}
```
Comment (bthaile) [inline packages/augur-ui/src/modules/reporting/common.styles.less:28]: how about this CSS, is there a more concise way?

Comment (phoebemirman) [reply packages/augur-ui/src/modules/reporting/common.styles.less:28]: you don't need text-align: center;
might not end up needing width: 100%; but not sure

File: packages/augur-ui/src/modules/reporting/components/reporting-report-markets/reporting-report-markets.tsx [hunk 2]
```diff
@@ -15,6 +18,12 @@ class ReportingReporting extends React.Component {
         <Helmet>
           <title>Reporting: Markets</title>
         </Helmet>
+        <div>
+          <ReportingModalButton
+            labelText="Designated Reporting Quick Guide"
```
Comment (phoebemirman) [inline packages/augur-ui/src/modules/reporting/components/reporting-report-markets/reporting-report-markets.tsx]: maybe this should be called "text" matches the other buttons

**Review by phoebemirman — APPROVED** [review]
looks good

File: packages/augur-ui/src/modules/reporting/components/reporting-report-markets/reporting-report-markets.tsx (hunk 2 above)
Comment (bthaile) [reply packages/augur-ui/src/modules/reporting/components/reporting-report-markets/reporting-report-markets.tsx]: updated property name to `text`

File: packages/augur-ui/src/modules/reporting/common.styles.less (hunk 1 above)
Comment (bthaile) [reply packages/augur-ui/src/modules/reporting/common.styles.less:28]: remove`text-align`

**Review by ghost — COMMENTED** [review]
This pull request has been automatically deployed to [FeaturePeek](https://featurepeek.com). 💜

 This link will always point to this PR's latest changes: https://peek.run/1rehx62

**Pull request merged**